yarn.lock

```diff
@@ -1175,6 +1175,11 @@ acorn@^8.1.0:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-8.1.0.tgz#52311fd7037ae119cbb134309e901aa46295b3fe"
   integrity sha512-LWCF/Wn0nfHOmJ9rzQApGnxnvgfROzGilS8936rqN/lfcYkY9MYZzdMqN+2NJ4SlTc+m5HiSa+kNfDtI64dwUA==
 
+acorn@^8.3.0:
```

Comment:
Why do we still have `acorn@^8.1.0` above? Does another dependency need to be updated?

Reply:
@guybedford Oh seems there is a git conflict with yarn.lock

Reply:
Thanks, fixed.